ArduPlane/quadplane.cpp

```diff
@@ -3524,6 +3524,9 @@ bool QuadPlane::using_wp_nav(void) const
     if (plane.control_mode == &plane.mode_qrtl && poscontrol.get_state() >= QPOS_POSITION2) {
         return true;
     }
+    if (plane.control_mode == &plane.mode_auto && poscontrol.get_state() > QPOS_APPROACH) {
+        return true;
+    }
```

Comment:
Not really your problem. But I wonder why were not just using `quadplane.wp_nav->is_active()`

Reply:
Looks like is only reporting stuff in anycase, I would vote for removing this whole function. Don't have to be in this PR tho.

Reply:
I think wp_nav can consider itself active even if we're not using it.  Perhaps only momentarily.